.travis.yml

```diff
@@ -1,47 +1,61 @@
+# Ubuntu 14.04 Trusty support
+sudo: required
```

Comment:
unfortunately OpenMPI and PETSc installation requires `apt-get install` because the packages are not in their white list (as far as I tested)